File: app/src/main/java/com/jgm90/cloudmusic/tables/PlaylistsTable.java
package com.jgm90.cloudmusic.tables;

import android.database.sqlite.SQLiteDatabase;

public class PlaylistsTable {

    public static String TABLE_NAME = "playlists";
    public static String COL_ID = "playlist_id";
    public static String COL_NAME = "name";
    public static String COL_OFFLINE = "offline";

    public static void create(SQLiteDatabase db) {
        db.execSQL("CREATE TABLE " + TABLE_NAME + "(" +
                COL_ID + " INTEGER PRIMARY KEY," +
                COL_NAME + " TEXT UNIQUE," +
                COL_OFFLINE + " NUMERIC);");
    }
}

File: app/src/main/java/com/jgm90/cloudmusic/tables/PlaylistsTable.kt
package com.jgm90.cloudmusic.tables

import android.database.sqlite.SQLiteDatabase

object PlaylistsTable {
    const val TABLE_NAME = "playlists"
    const val COL_ID = "playlist_id"
    const val COL_NAME = "name"
    const val COL_OFFLINE = "offline"

    fun create(db: SQLiteDatabase) {
        db.execSQL(
            "CREATE TABLE $TABLE_NAME(" +
                "$COL_ID INTEGER PRIMARY KEY," +
                "$COL_NAME TEXT UNIQUE," +
                "$COL_OFFLINE NUMERIC);"
        )
    }
}
